frontend/website/docs/architecture/lifecycle-payment-technical.md

```diff
@@ -0,0 +1,216 @@
+# The Lifecycle of a Payment - Technical Guide
+
+In Coda, payments pass through several steps before they are considered verified and complete. This document is meant to walk through what happens to a single payment as it works its way through the codebase. For a more high-level overview aimed at users who want to understand how payments work on a macroscopic level, check out the [lifecycle of a payment](/docs/architecture/lifecycle-payment).
+
+Let's say you want to send a payment in Coda, assuming you've already created an account and have received funds.
+Your friend gives you her public key -- it's `KEFLx5TOqJNzd6buc+dW3HCjkL57NjnZIaplYJ50DO1uTfogKfwAAAAA`.
+
+Then you invoke the following command:
+
+```bash
+$ coda client send-payment \
+    -amount 10 \
+    -receiver KEFLx5TOqJNzd6buc+dW3HCjkL57NjnZIaplYJ50DO1uTfogKfwAAAAA \
+    -fee 3 \
+    -privkey-path keys/my-wallet
+```
+
+Where:
+- `amount` is the amount of coda you are sending
+- `receiver` is the public key of the intended recipient
+- `fee` is the fee to be paid to the network to process the transaction
+- `privkey-path` is the filepath to the private key for your account
+
+<a name="client"></a>
```

Comment:
did you manually add these anchor tags?

Reply:
no, this is leftover from the previous doc - I'll go ahead and remove it